app/views/layouts/application.html.erb

```diff
@@ -11,28 +11,24 @@
     <%= javascript_include_tag "application" %>
   </head>
   <body>
-
-    <%= nav_bar brand: 'Zartan', static: :top, responsive: true do %>
-      <%= menu_group do %>
-        <%= menu_item "Sites", sites_path %>
-        <%= menu_item "Sources", sources_path %>
-        <%= menu_item "Settings", config_path %>
+    <% if current_user %>
+      <%= nav_bar brand: 'Zartan', static: :top, responsive: true do %>
+        <%= menu_group do %>
+          <%= menu_item "Sites", sites_path %>
+          <%= menu_item "Sources", sources_path %>
+          <%= menu_item "Settings", config_path %>
+          <%= menu_item "Sign out", signout_path %>
+        <% end %>
       <% end %>
     <% end %>
-
     <div class="container">
       <div class="row">
         <div class="col-md-12">
+          <% unless current_user %><br/><% end %>
           <%= bootstrap_flash %>
           <%= yield %>
         </div>
       </div><!--/row-->
-
-      <footer>
-        <p>&copy; Raybeam 2015</p>
```

Comment:
why keep it...it is hardcoded as a date. I was going to do a UI redesign after I spoke with David as well, so I just removed the copyright here.